modules/ROOT/nav.adoc

```diff
@@ -1,11 +1,11 @@
 .xref:index.adoc[About Anypoint Platform Private Cloud Edition]
 * xref:index.adoc[About Anypoint Platform Private Cloud Edition]
-* xref:supported-cluster-config.adoc[About Supported Network Topologies for Anypoint Private Cloud Edition]
-* xref:prereq-workflow.adoc[Workflow: Install and Verify Prerequisites for Physical Servers]
- ** xref:prereq-hardware.adoc[About Private Cloud Edition Hardware Requirements]
- ** xref:prereq-network.adoc[About Private Cloud Edition Network Requirements]
- ** xref:prereq-verify-disk.adoc[To Verify Hardware Requirements]
- ** xref:prereq-verify-device.adoc[To Verify Anypoint Private Cloud Device Requirements]
+* xref:supported-cluster-config.adoc[Supported Network Topologies]
+* xref:prereq-workflow.adoc[Workflow: Install and Verify Prerequisites]
+ ** xref:prereq-hardware.adoc[Hardware Requirements]
+ ** xref:prereq-verify-disk.adoc[To Verify Disk Requirements]
+ ** xref:prereq-verify-device.adoc[To Verify Device Requirements]
+ ** xref:prereq-network.adoc[Network Requirements]
```

Comment:
Hardware > Verify Disk > Verify Device should be grouped, separate from network requirements.

Removed some text that make the left pane links to be multiple lines and very long